lib/potluck.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require './lib/dish'
+
+# Potluck Class
+class Potluck
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu = {
+      appetizer: [],
+      entree: [],
+      dessert: []
+    }
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    @dishes.find_all do |dish|
+      dish.category == category
+    end
+  end
+
+  def menu
```

Comment:
This method looks great, but be careful about naming a method the same as an instance variable!